Don't include openssl EPIPE complaints in input.

--- pexpect-4.6.0/tests/test_expect.py.orig	2018-04-11 21:03:57.000000000 +0200
+++ pexpect-4.6.0/tests/test_expect.py	2018-08-01 16:18:13.739732629 +0200
@@ -411,7 +411,7 @@
     def test_before_across_chunks(self):
         # https://github.com/pexpect/pexpect/issues/478
         child = pexpect.spawn(
-            '''/bin/bash -c "openssl rand -base64 {} | head -500 | nl --number-format=rz --number-width=5 2>&1 ; echo 'PATTERN!!!'"'''.format(1024 * 1024 * 2),
+            '''/bin/bash -c "openssl rand -base64 {} 2>/dev/null| head -500 | nl --number-format=rz --number-width=5 ; echo 'PATTERN!!!'"'''.format(1024 * 1024 * 2),
             searchwindowsize=128
         )
         child.expect(['PATTERN'])
